lib/spack/docs/packaging_guide.rst

```diff
@@ -515,52 +514,36 @@ of its versions, you can add an explicit URL for a particular version:
 
 
 When you supply a custom URL for a version, Spack uses that URL
-*verbatim* and does not perform extrapolation.
+*verbatim* and does not perform extrapolation. The order of precedence
+of these methods is:
 
-^^^^^^^^^^^^^^^^^^^^^
-PyPI and version URLs
```

Comment:
This section has been moved to http://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#url.